src/pages/home/IncidentCard.tsx

```diff
@@ -58,30 +70,41 @@ type Props = {
   incident: IncidentInterface
 }
 const IncidentCard = (props: Props) => {
+  const [index, setIndex] = useState('')
```

Comment:
I would simply have a boolean value `isExpanded` and set its default value to `status !== 'stable'`
To update it you can simply do `setIsExpanded(state => !state)`